### PR TITLE
Introduce FileLogger

### PR DESCRIPTION
**Pull request by thejoshuahendrix**
This is the mvp for the file logging feature. it succesfully writes to an outside file, what we need to do from here is integrate the function into logger, i commented out the date part of the filename for testing, in prod you would remove that comment and have the date in the log title, most likely followed by the time.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (lucemans) [inline src/index.ts:13]: Lol

Comment (Antony1060) [inline src/index.ts:13]: Yep, I agree. This specific part of code is probably the most important one

Comment (thejoshuahendrix) [reply src/index.ts:13]: very true

**Review by lucemans — APPROVED** [review]
LGTM

**Pull request merged**